pam/integration-tests/testdata/tapes/cli/qr_code.tape

```diff
@@ -4,7 +4,7 @@ Output qr_code.gif # If we don't specify a .gif output, it will create a default
 # Configuration header to standardize the output.
 # Does not work with the "Source" command.
 Set Width 800
-Set Height 600
```

Comment:
Why was this necessary? We only have a word change, we should not need to increase the height.

Reply:
See this commit: https://github.com/ubuntu/authd/pull/408#pullrequestreview-2154767767

Reply:
Ouch, wrong paste: https://github.com/ubuntu/authd/pull/408/commits/37f6d560e115a85a78495a18f56fa5629b139f73